proposals/0029-cooperative-vector.md

```diff
@@ -789,7 +789,7 @@ if (TierSupport.CooperativeVectorTier >= D3D12_COOPERATIVE_VECTOR_TIER_1_0) {
 
     // Use CheckFeatureSupport call to query only MatrixVectorMulAddProperties
     UINT MatrixVectorMulAddPropCount = CoopVecProperties.MatrixVectorMulAddPropCount;
-    std::vector<D3D12_COOPERATIVE_VECTOR_PROPERTIES_INFERENCE> properties(MatrixVectorMulAddPropCount);
+    std::vector<D3D12_COOPERATIVE_VECTOR_PROPERTIES_MUL> properties(MatrixVectorMulAddPropCount);
```

Comment:
The comments above and below this say "inference" intrinsic.  Should change to mul